Angular: Add feature toggle to disable angular at runtime (#65245)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

| `prometheusAzureOverrideAudience` | Experimental. Allow override default AAD audience for Azure Prometheus endpoint                                                                                                        |
| `migrationLocking`                | Lock database during migrations                                                                                                                                                        |
| `newDBLibrary`                    | Use jmoiron/sqlx rather than xorm for a few backend services                                                                                                                           |
| `validateDashboardsOnSave`        | Validate dashboard JSON POSTed to api/dashboards/db                                                                                                                                    |
| `autoMigrateOldPanels`            | Migrate old angular panels to supported versions (graph, table-old, worldmap, etc)                                                                                                     |

> | `disableAngular`                  | Dynamic flag to disable angular at runtime. The prefered method is to set `angular_support_enabled` = false in the [security] settings, but this allows changing the state at runtime. |

Reviewer: ```suggestion
| `disableAngular`                  | Dynamic flag to disable angular at runtime. The preferred method is to set `angular_support_enabled` to `false` in the [security] settings, which allows you to change the state at runtime. |
```
